dotnet-source/ks.fiks.io.arkivsystem.sample/Handlers/BaseHandler.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Xml.Schema;
+using KS.Fiks.ASiC_E;
+using KS.Fiks.IO.Client.Models;
+using Serilog;
+
+namespace ks.fiks.io.arkivsystem.sample.Handlers
+{
+    public class BaseHandler
+    {
+        private static readonly ILogger Log = Serilog.Log.ForContext(MethodBase.GetCurrentMethod()?.DeclaringType);
+
+        protected static string GetPayloadAsString(MottattMeldingArgs mottatt, XmlSchemaSet xmlSchemaSet,
```

Comment:
Ville opprettet et objekt `ValidationResult` som inneholder resultat, warning/error og isSuccess for å få vekk brukt av `out`